### PR TITLE
feat: add "allowLocalProjects" switch

### PR DESCRIPTION
**Pull request by madetara**
May be useful when publishing multiple dependent packages from same solution and want their versions to be synced

### CODE REVIEW

File: dotnetcementrefs/dotnetcementrefs.cs [hunk 1]
```diff
@@ -186,12 +186,12 @@ private static bool ShouldIgnore(Project project)
...
 
     private static ProjectItem[] FindCementReferences(Project project, ISet<string> localProjects,
-        string[] cementRefPrefixes)
+        string[] cementRefPrefixes, bool allowLocalProjects)
     {
         return project.Items
             .Where(item => item.ItemType == "Reference")
             .Where(item => cementRefPrefixes.Any(x => item.EvaluatedInclude.StartsWith(x)))
-            .Where(item => !localProjects.Contains(item.EvaluatedInclude))
+            .Where(item => allowLocalProjects || !localProjects.Contains(item.EvaluatedInclude))
```
Comment (KungA) [inline dotnetcementrefs/dotnetcementrefs.cs]: Are you sure, that `allowLocalProjects` should be used inside `FindCementReferences`?
Can we just add them later?

Comment (madetara) [reply dotnetcementrefs/dotnetcementrefs.cs]: Sounds legit. 

I also thought that maybe we should explicitly specify local projects to resolve

Comment (madetara) [reply dotnetcementrefs/dotnetcementrefs.cs]: Something like `--includeLocal ProjectA ProjectB`

Comment (KungA) [reply dotnetcementrefs/dotnetcementrefs.cs]: Seems better to exclude some projects later (in super special cases).

Comment (KungA) [reply dotnetcementrefs/dotnetcementrefs.cs]: And it should be very similar with `ReferencesToRemove`.

**Comment by madetara** [comment]
Moved local projects adding to separate method.
It slightly looks like DRY violation, but i preferred to keep it clean, since it's more valuable than efficiency, from our previous discussion.

Also bumped version for tool to update properly

**Pull request merged**